Sources/ComposableArchitecture/Documentation.docc/Articles/MigratingToTheReducerProtocol.md

```diff
@@ -460,18 +460,18 @@ Reducer { state, action, environment in
 ```
```

Comment:
I did not update the old `bindable state` keywords in this specific part because we had used `@BindableState` before 'ReducerProtocol' came out. But I'm totally open to suggestions on this. Just let me know your thoughts.